spec/rails_helper.rb

```diff
@@ -5,6 +5,8 @@
 # Prevent database truncation if the environment is production
 abort("The Rails environment is running in production mode!") if Rails.env.production?
 require 'rspec/rails'
+require 'support/factory_bot'
```

Comment:
Please put it below `# Add additional requires below this line. Rails is not loaded until this point!`